cebra/models/layers.py

```diff
@@ -97,3 +97,25 @@ def forward(self, inp: torch.Tensor) -> torch.Tensor:
         connect = self.layer(inp)
         downsampled = F.interpolate(inp, scale_factor=1 / self.downsample)
         return torch.cat([connect, downsampled[..., :connect.size(-1)]], dim=1)
+
+
+class _SkipLinear(nn.Module):
```

Comment:
when is this used?

Reply:
in MLP based models where we want to use skip connections, e.g. here

![image](https://github.com/user-attachments/assets/4131fc4f-a045-4217-a87e-91f7ae1bf4ba)